src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Random random = new Random();

```

Comment:
create class-level variables for random and colorSupplier to avoid creating these objects each time when you call getRandomBall() method